src/calculateRentalCost.js

```diff
@@ -4,7 +4,21 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const LONG_TERM = 7;
+  const LONG_TERM_DISCOUNT = 50;
+  const SHORT_TERM = 3;
+  const SHORT_TERM_DISCOUNT = 20;
+  const DAY_PRICE = 40;
+
+  if (days < SHORT_TERM) {
+    return days * DAY_PRICE;
+  }
+
+  if (days < LONG_TERM) {
+    return days * DAY_PRICE - SHORT_TERM_DISCOUNT;
```

Comment:
There is a logical issue here. The discount for short-term rentals is applied regardless of the number of days. According to the logic, the discount should only apply if the number of days is greater than or equal to SHORT_TERM but less than LONG_TERM. Consider changing the condition to `if (days >= SHORT_TERM && days < LONG_TERM)`.

Reply:
There is not a logical issue here.  Because if days < SHORT_TERM I return  right value before.